prep-javascript-dom-events/index.html

```diff
@@ -0,0 +1,20 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>JavaScript Events</title>
+</head>
+
+<body>
+  <button class="hello-world-btn">Hello World!</button>
+  <script>
+    document.addEventListener('DOMContentLoaded', console.log('content loaded!'));
+    window.addEventListener('mousemove', console.log('mouse is moving!'));
+    var $helloWorld = document.querySelector('button.hello-world-btn');
+    $helloWorld.addEventListener('click', console.log('hello world!'));
```

Comment:
It's important to wrap these console logs in function definitions so that they can possibly run more than once

```js
window.addEventListener('mousemove', function () {
  console.log('mouse is moving');
});
```